_posts/2018-01-14-hackathon.md

```diff
@@ -39,13 +39,13 @@ but please make sure your team does not have more than 4 members.
 
 * **aTeam**
 
-  **Nimit Bhardwaj(3rd)**
+  **Nimit Bhardwaj (3rd) / Udit Gulati (2nd) / Amit Yadav (2nd)**
 
   c++, python
 
 * **#define true false**
 
-  **Anishka Gupta(3rd) / Sarbjit(3rd) / jatin nagal(1st)**
+  **Anishka Gupta(3rd) / Sarbjit(3rd) / jatin nagpal(1st)**
```

Comment:
Should have capitalised the name too I guess :)

Reply:
Yeah, but this is fine too as long as the name is correct.

Reply:
Eh, some people care about name capitalizations and it adds a shine of correctness to the blog. I was just mentioning it for next time.

Reply:
Well, there are a few other typos in this post. I think the author of this post @vishichoudhary  should fix them ASAP :P.